pyproject.toml

```diff
@@ -49,6 +49,7 @@ issues = "https://github.com/instructlab/instructlab/issues"
 
 [project.entry-points."instructlab.command.data"]
 "generate" = "instructlab.data.generate:generate"
+"list" = "instructlab.data.list:list_data"
```

Comment:
(No action required) nit: model list has `model_list` function, but here you have `list_data`. Maybe change to `data_list` for consistency?

Reply:
Sure, I'll make this change.